python/graphstorm/gconstruct/utils.py

```diff
@@ -530,6 +541,117 @@ def to_tensor(self):
         """
         return th.tensor(self.to_numpy())
 
+class ExtFeatureWrapper(ExtMemArrayWrapper):
+    """ The wrapper to memory-mapped Numpy array when combining feature
+
+    Parameters
+    ----------
+    arr_path : str
+        The path to different feature files directory
+    shape : tuple
+        The shape of the array.
+    dtype : numpy dtype
+        The data type.
+    merged_file: str
+        The merged file name
+    """
+    def __init__(self, arr_path, shape, dtype,
+                 merged_file="merged_feature.npy"):
+        self._arr_path = arr_path
+        self._shape = (shape[0], 0)
+        self._orig_dtype = self._dtype = dtype
+        self._arr = None
+        self.merged_file = merged_file
+        self.wrapper = []
+
+    @property
```

Comment:
Can we inherit from ExtNumpyWrapper? Many functions are same. And this class can be treated as a sub-class of ExtNumpyWrapper as it also uses Numpy mmap.

Reply:
The only concern I did not inherit is about the shape definition here. Let me check if we can do so. Mainly about the shape.